### PR TITLE
Add localConfig instructions.

### PR DESCRIPTION
**Pull request by aljones15**

### CODE REVIEW

**Review by tminard — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "mocha tests/ --reporter @digitalbazaar/mocha-w3c-interop-reporter --reporter-options abstract=\"$PWD/config/abstract.hbs\",reportDir=\"$PWD/reports\",respec=\"$PWD/config/respec.json\",suiteLog='./suite.log',templateData=\"$PWD/reports/index.json\",title=\"Data Integrity eddsa 2022 Interoperability Report 1.0\" --timeout 15000 --preserve-symlinks",
-    "lint": "eslint ."
+    "lint": "eslint --ext .js,.cjs ."
```
Comment (tminard) [inline package.json:12]: Is there an eslint config file? These can go there

File: README.md [hunk 2]
```diff
@@ -86,47 +86,47 @@ Content-Type: application/ld+json
...
-After adding the config file, both the localhost implementations and other
-implementations matching the test tag will be included in the test run.
-
-To specifically test only the localhost implementation, modify the test suite to
-filter implementations based on a specific tag in your local configuration file.
-
-For instance, if your `.localImplementationsConfig.cjs` config file looks like
-the config above, you can adjust the tag used in each test suite by modifying `./config/runner.json`
-to filter the implementations by `localhost` and other tags.
+To specifically test only the implementations in `localConfig.cjs`, change `settings.testAllImplementations` to `false`.
```
Comment (BigBlueHat) [inline README.md:129]: Line's a bit long.

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
Fiddly code formatting tweak request.

File: package.json (hunk 1 above)
Comment (aljones15) [reply package.json]: this kind of surprised me, but actually I don't think there is a config file way of specifying the file extensions, but there is an eslint config file way of specifying the file glob patterns so I used that here: https://github.com/w3c/vc-di-eddsa-test-suite/pull/46/commits/40e0733bf692bed79f28badff9ef29dfa4843e75

File: README.md (hunk 2 above)
Comment (BigBlueHat) [reply README.md]: Looks fine to me. Thanks.

**Pull request merged**